.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
is this necessary? I think we have babel config inlined in our `webpack.config.js` and having two copies means having to update two places if we change this.